Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -231,6 +393,24 @@ public bool IsAdmin(string username)
         {
             return _context.Participant.Any(p => p.Username == username && p.IsAdmin == true);
         }
+
+        private string GetCustomUnqiueUsername(string username)
+        {
+            var customSuffix = ".custom";
```

Comment:
This should be a const string. It should probably be in Names.cs, or at least a public const member of this class.

```csharp
public const string CUSTOM_PARTICIPANT_USERNAME_SUFFIX = ".custom";
```

Reply:
@Kevin-Chu58